Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'jbuilder', '~> 2.7'
 # gem 'redis', '~> 4.0'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+gem 'faker', :git => 'https://github.com/faker-ruby/faker.git', :branch => 'master'
```

Comment:
" :git => 'https://github.com/faker-ruby/faker.git', :branch => 'master' "
I don't understand this part, what does it fo?